docs/en/sql-reference/10-sql-commands/30-query-operators/json.md

```diff
@@ -17,4 +17,6 @@ import FunctionDescription from '@site/src/components/FunctionDescription';
 | @> | Checks if the left JSON expression contains all key-value pairs of the right JSON expression, returning 1 for true and 0 for false. | `PARSE_JSON('{"name":"Alice","age":30}') @> PARSE_JSON('{"name":"Alice"}')` | 1 |
```

Comment:
We've changed the casting rule from string to variant, so the example can be written more simply like this
```sql
'{"name":"Alice","age":30}'::json @> '{"name":"Alice"}'::json
```